docs/UserManuals/TeamConfiguration.md

```diff
@@ -62,7 +62,7 @@ b. The actual api request.
 If there is a problem with team_users association or data in users, just re-put users api interface, i.e. (b in step 2 above)
 
 ## Step 4 - Collect accounts 
```

Comment:
Good writeup here. Are you constructing the fake account data by running SQL Insert statements? I think it'll help to also include them here to save the reader a couple of minutes.

Reply:
yes, I will add it, thank you.

Reply:
updated